Cargo.toml

```diff
@@ -2,187 +2,193 @@
-  "external-crates/move/crates/module-generation",
-  "external-crates/move/crates/move-abstract-stack",
-  "external-crates/move/crates/move-analyzer",
-  "external-crates/move/crates/move-binary-format",
-  "external-crates/move/crates/move-borrow-graph",
-  "external-crates/move/crates/move-bytecode-source-map",
-  "external-crates/move/crates/move-bytecode-utils",
-  "external-crates/move/crates/move-bytecode-verifier",
-  "external-crates/move/crates/move-bytecode-viewer",
-  "external-crates/move/crates/move-cli",
-  "external-crates/move/crates/move-command-line-common",
-  "external-crates/move/crates/move-compiler",
-  "external-crates/move/crates/move-compiler-transactional-tests",
-  "external-crates/move/crates/move-core-types",
-  "external-crates/move/crates/move-coverage",
-  "external-crates/move/crates/move-disassembler",
-  "external-crates/move/crates/move-docgen",
-  "external-crates/move/crates/move-errmapgen",
-  "external-crates/move/crates/move-explain",
-  "external-crates/move/crates/move-ir-compiler",
-  "external-crates/move/crates/move-ir-compiler-transactional-tests",
-  "external-crates/move/crates/move-ir-to-bytecode",
-  "external-crates/move/crates/move-ir-to-bytecode-syntax",
-  "external-crates/move/crates/move-ir-types",
-  "external-crates/move/crates/move-model",
-  "external-crates/move/crates/move-package",
-  "external-crates/move/crates/move-proc-macros",
-  "external-crates/move/crates/move-prover",
-  "external-crates/move/crates/move-prover-test-utils",
-  "external-crates/move/crates/move-read-write-set-types",
-  "external-crates/move/crates/move-stackless-bytecode",
-  "external-crates/move/crates/move-stackless-bytecode-interpreter",
-  "external-crates/move/crates/move-stdlib",
-  "external-crates/move/crates/move-symbol-pool",
-  "external-crates/move/crates/move-transactional-test-runner",
-  "external-crates/move/crates/move-unit-test",
-  "external-crates/move/crates/move-vm-config",
-  "external-crates/move/crates/move-vm-integration-tests",
-  "external-crates/move/crates/move-vm-profiler",
-  "external-crates/move/crates/move-vm-runtime",
-  "external-crates/move/crates/move-vm-test-utils",
-  "external-crates/move/crates/move-vm-transactional-tests",
-  "external-crates/move/crates/move-vm-types",
-  "external-crates/move/crates/serializer-tests",
-  "external-crates/move/crates/test-generation",
-  "external-crates/move/move-execution/v0/move-bytecode-verifier",
-  "external-crates/move/move-execution/v0/move-stdlib",
-  "external-crates/move/move-execution/v0/move-vm/runtime",
-  "external-crates/move/move-execution/v1/crates/move-bytecode-verifier",
-  "external-crates/move/move-execution/v1/crates/move-stdlib",
-  "external-crates/move/move-execution/v1/crates/move-vm-runtime",
-  "sdk/move-bytecode-template",
+    "external-crates/move/crates/bytecode-interpreter-crypto",
+    "external-crates/move/crates/bytecode-interpreter-testsuite",
```

Comment:
nit, spurious whitespace changes

Reply:
If I had to guess, this is the doing of the execution layer script? (Not sure what has caused it to have a change of heart about the indent level here though)

Reply:
fixed